Illumio - Fix Invalid IP List HREF (#31921)
demisto/content: Packs/Illumio/ReleaseNotes/1_0_18.md

#### Integrations

##### Illumio Core

- Updated the Docker image to: *demisto/illumio:1.0.0.83370*.

#### Playbooks

##### Rapid ransomware containment - Illumio

> - Fix hardcoded IP list HREF in Rapid ransomware containment playbook

Reviewer: ```suggestion
- Fixed an issue where the playbook included a hardcoded IP list HREF.
```
